css/help_dialog.css

```diff
@@ -1,12 +1,5 @@
-#help-dialog li a[href="#create_ticket"] span[class="text"] {
-     display: none !important;
+/* hide the help menu - Note it will be un-hidden by javascript */
+#ui-id-2 {
```

Comment:
This selector is too broad - this is the one that was causing the Sections tab on the Settings page to be hidden (and it might affect other things on other pages).